compiler+runtime/src/cpp/jank/runtime/context.cpp

```diff
@@ -44,6 +44,7 @@ namespace jank::runtime
     /* We want to initialize __rt_ctx ASAP so other code can start using it. */
     : binary_version{ (__rt_ctx = this, util::binary_version()) }
     , binary_cache_dir{ util::binary_cache_dir(binary_version) }
+    , opts{ util::cli::opts }
```

Comment:
We should not be copying the opts.

The issue wasn't that we were never initializing the copy of the opts. The issue was that we had a copy of the opts in the first place.